content/manuals/enterprise/security/hardened-desktop/settings-management/configure-json-file.md

```diff
@@ -258,6 +258,7 @@ The following tables describe all available settings in the `admin-settings.json
 | `displayedOnboarding` |  | If `value` is set to `true`, the onboarding survey will not be displayed to new users. Setting `value` to `false` has no effect. |  Docker Desktop version 4.30 and later |
 | `desktopTerminalEnabled` |  | If `value` is set to `false`, developers cannot use the Docker terminal to interact with the host machine and execute commands directly from Docker Desktop. |  |
 |`exposeDockerAPIOnTCP2375`| Windows only| Exposes the Docker API on a specified port. If `value` is set to true, the Docker API is exposed on port 2375. Note: This is unauthenticated and should only be enabled if protected by suitable firewall rules.|  |
+| `silentModulesUpdate` | | If `value` is set to `true`, Docker Desktop automatically updates components that don't require a restart. For example, the Docker CLI or Docker Scout components. | Docker Desktop version 4.46 and later. |
```

Comment:
When I created the setting, I hadn’t realized it would be publicly visible.
Perhaps we should have named it `automaticComponentsUpdate`. 🤔 

@KatTomrushka what do you think? Is this a problem we need to fix?